docs/developer-guides/framework/hub-integration-and-caching.md

```diff
@@ -15,17 +15,17 @@ Towhee hub is deployed on a Gitea server. Gitea is an open-source, self-hosted g
 
 **Integration**
 
-All of Towhees communication with the hub is done through http requests. This was done in order to avoid requiring git installations on the system. 
+All of Towhee's communication with the hub is done through http requests. This was done in order to avoid requiring git installations on the system. 
 
-Towhee operations that interact with the hub can be split into two categories, those that require authorization and those that do not. Lets begin with those that do not. 
+Towhee operations that interact with the hub can be split into two categories, those that require authorization and those that do not. Lets begin with those that do not.
 
 **Non-Authorization**
 
 The main operation that does not require authorization is downloading the repo. This request works by first obtaining the extensions that are stored in lfs. The next step is to download all the file names and group them by lfs and non-lfs downloads. This is required since LFS requires using a different http call to download the full data. Once we have these groups and file names, we can perform a multithreaded download.
 
 **Authorized Requests**
 
-Authorized requests require that you have login credentials setup with our hub. The main operations that require authorization and creating and deleting a repo. These requests use temporary tokens to speed up execution. 
+Authorized requests require that you have login credentials setup with our hub. The main operations that require authorization and creating and deleting a repo. These requests use temporary tokens to speed up execution.
```

Comment:
*are* creating
These requests use temporary tokens to speed up execution. -> might need to change soon^tm